smarts/sstudio/genscenario.py

```diff
@@ -403,7 +409,28 @@ def _validate_entry_tactic(mission):
             ), f"Zone edge `{z_edge}` is not the same edge as `types.Mission` route begin edge `{edge}`"
 
 
-def gen_traffic_histories(scenario: str, histories):
-    output_path = os.path.join(scenario, "traffic_histories.pkl")
-    with open(output_path, "wb") as f:
-        pickle.dump(histories, f)
+def gen_traffic_histories(scenario: str, histories_datasets, overwrite: bool):
+    genhistories_py = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "genhistories.py"
+    )
+    for hdsr in histories_datasets:
```

Comment:
I am wondering what does `hdsr` and `hds` stand for?...

Reply:
hds -> "history data set" and hdsr -> "history data set ref" (or something like that!)